vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go

```diff
@@ -0,0 +1,146 @@
+package test
```

Comment:
Why this file wasn't imported before and now it's completely new? Do we really need it?